proposals/0242-static-nonce-account.md

```diff
@@ -0,0 +1,73 @@
+---
+simd: '0242'
+title: Static Nonce Account Only
+authors:
+  - Andrew Fitzgerald (Anza)
+category: Standard
+type: Core
+status: Review
+created: 2025-02-03
+feature:
+---
+
+## Summary
+
+The advance nonce instruction in a transaction can currently specify any
+account in the transaction as the account to advance.
+This proposal would restrict the advance nonce instruction to only be able to
+advance a statically included account.
+
+## Motivation
+
+- There is a separate proposal to relax the constraints on transaction account
+  resolution. That proposal would allow transactions with invalid lookups or
+  duplicate accounts to be included in a block, pay fees, but not processed.
+- However, that relaxation cannot happen if the nonce account is advanced in a
+  looked up account.
```

Comment:
Can you explain further why the relaxation cannot happen in this case? It's not too obvious I think. My understanding is that if we attempt to process a durable nonce transaction that uses an invalid lookup, we wouldn't be able to include the transaction because the nonce is not advanceable and therefore there wouldn't be a way to ensure that it would only be processable once. This is a blocker to implementing the other proposal which aims to allow including any transaction which has an invalid lookup into a block.

Reply:
it also sort of resolves a catch-22: any reasonable definition of nonce validation would fail if it couldnt be resolved, and we would still need to validate age of blockhash transactions for failed alt resolutions, so the only possible way we could "process" a failed lookup nonce transaction is to pretend its a blockhash transaction _and_ skip blockhash validation for it

a simpler solution would just be to clarify in simd192 that failed nonce lookup == failed nonce validation. but im inclined to support simd242 because im in favor of restricting nonces in this way (and several others)

Reply:
i think it makes sense to elaborate the motivation so these details are explicit